docs/library/ubluetooth.rst

```diff
@@ -313,33 +331,35 @@ Central Role (GATT Client)
     Returns ``False`` if the connection handle wasn't connected, and ``True``
     otherwise.
 
-.. method:: BLE.gattc_discover_services(conn_handle)
+.. method:: BLE.gattc_discover_services(conn_handle, [uuid])
 
     Query a connected peripheral for its services.
 
+    Optionally specify
+
```

Comment:
Documentation of `uuid` incomplete/missing (also below)

Reply:
Thanks, will fix.